packages/@aws-cdk/cloudformation-diff/lib/iam/statement.ts

```diff
@@ -1,6 +1,9 @@
-import * as deepEqual from 'fast-deep-equal';
 import { deepRemoveUndefined } from '../util';
 
+// namespace object imports won't work in the bundle for function exports
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const deepEqual = require('fast-deep-equal');
```

Comment:
I wonder if we could flip to use https://nodejs.org/api/util.html#utilisdeepstrictequalval1-val2 ? (Possibly the answer is no, I'm not certain the semantics are the same, but from a cursory look, they might be).

Reply:
Perhaps. I kind of don't want to rock the boat to much here though. I'll leave it as is.